waspc/examples/todoApp/.gitignore

```diff
@@ -6,6 +6,15 @@ node_modules/
 .env
 .env.*
 
+# These two we added only because dotenv-vault keeps adding them if it doesn't find them,
+# even though we don't need them. Remove them once dotenv-vault stops doing that.
+.env*
+.flaskenv*
+
 # Don't ignore example dotenv files.
 !.env.example
 !.env.*.example
+
+# These are config files for dotenv-vault, so we don't want to ignore them.
+!.env.project
+!.env.vault
```

Comment:
Missing empty line.


Also, same questions wrt the template as in the other gitignore comment.

Reply:
So recently I updated the default gitignore. In that PR I also udpated all the tempaltes and example apps, but I missed headless-test/examples/todoApp/.gitignore. So I updated it in this PR, that was the other comment you had, I reponded to that one.

As for this specific comment:
1. Hm, I deledeted the empty line here? Was there ever one? SHould we have one? I guess it is good to have one for the diffing purpose? Any other reason? Could you elaborate a bit, what is your opinion on this?
2. Here I wasn't updateing .gitignore, it was already up to date with the template. But I had to add these 4 entries (in 2 groups) because of the vault-dotenv that we are using. So this is specific to this project.

Reply:
The empty line thing - hmm, I'm not sure what you're asking.

GitHub flags files that have a last "newline" symbol missing with that "invalid direction" traffic sign, I was just pointing that out:
![image](https://github.com/wasp-lang/wasp/assets/22752307/f983ec8d-9847-4559-bc16-329dc13d4469)

I guess this is because of the usual argument of "Linux tooling relies on lines ending with the newline symbol" and other misc reasons ([source](https://stackoverflow.com/a/5813359)).

But, since I know you know all of this, you're probably asking should we care?

I don't know, but since GitHub marks it as missing and all the formatters add it, I usually always make sure to add them myself.

Reply:
No I actually wasn't sure about it! Never payed attention to the ending newline. Ok, I will add it!